tests/test_GLM_single.py

```diff
@@ -0,0 +1,73 @@
+from os.path import join
+from os.path import abspath
+from os.path import dirname
+import scipy
+import scipy.io as sio
+import numpy as np
+from glmsingle.glmsingle import GLM_single
+
+test_dir = dirname(abspath(__file__))
+expected_dir = join(test_dir, "expected", "python")
+data_dir = join(test_dir, "data")
+data_file = join(data_dir, "nsdcoreexampledataset.mat")
+
+output_dir = join(test_dir, "outputs")
+
+
+expected = {"typeb": {}, "typec": {}, "typed": {}}
+
+# TODO use same expected data for Python and MATLAB ?
+# TODO in both cases expected results should probably be in a set of CSV files
+# tmp = sio.loadmat(join(expected_dir, "TYPEB_FITHRF.mat"))
+# expected["typeb"]["HRFindex"] = tmp["HRFindex"]
```

Comment:
In the long run we probably do not want to have different expected data for MATLAB and python, a set of CSV files to read from for both languages could be easier to handle, more interoperable and easier to inspect too.